turbo/execution/eth1/forkchoice.go

```diff
@@ -519,6 +520,10 @@ func (e *EthereumExecutionModule) updateForkChoice(ctx context.Context, original
 			}
 			gasUsedMgas := float64(fcuHeader.GasUsed) / 1e6
 			mgasPerSec := gasUsedMgas / totalTime.Seconds()
+			metrics.GetOrCreateCounter("chaintip_mgas_per_sec").Add(mgasPerSec)
+			metrics.GetOrCreateCounter("chaintip_flush_latency_ms").Add(float64(blockTimings[engine_helpers.BlockTimingsFlushExtendingFork].Milliseconds()))
```

Comment:
usually in Prometheus metrics are called `_seconds` - but they are actually nanoseconds
Example: https://prometheus.io/docs/prometheus/latest/querying/functions/
Example: 
```
blockExecutionTimer = metrics.GetOrCreateSummary("chain_execution_seconds")
blockExecutionTimer.ObserveDuration(time.Now())
```